test/tests.py

```diff
@@ -21,96 +22,124 @@ def setUp(self):
         self.assertEqual(resp.status_code, 201)
 
     def test_get_single_event(self):
         """Test that gets single event"""
-        resp = self.test_client().get('/api/events/event1')
+        resp = self.test_client().get('/api/events/1')
         self.assertEqual(resp.status_code, 200)
 
     def test_delete_event(self):
         """Test that event gets delete"""
-        resp = self.test_client().delete('/api/events/event2')
+        resp = self.test_client().delete('/api/events/2')
         self.assertEqual(resp.status_code, 204)
 
     def test_put_event(self):
         """Test that event get put"""
-        resp = self.test_client().put('/api/events/event1', data=self.eventCreate)
+        resp = self.test_client().put('/api/events/1', data=self.eventcreate)
         self.assertEqual(resp.status_code, 201)
 
+class TestRSVP(unittest.TestCase):
+    """Test event rsvp"""
+    def setUp(self):
+        self.app = app
+        self.rsvp = RSVP()
+        self.test_client = self.app.test_client
+        self.testrsvp = {
+            'name': u'John Doe',
+            'email': u'atony.nesh@gmail.com'
+        }
+        self.testemailexist = {
+            'name': u'John Doe',
+            'email': u'john.D@gmail.com'
+        }
+
+    def tearDown(self):
+        del self.rsvp
+        del self.test_client
+        del self.testrsvp
+        del self.testemailexist
+
+    def test_event_rsvp(self):
+        """Test user event rsvp"""
+        res = self.test_client().post('/api/events/1/rsvp', data=self.testrsvp)
+        self.assertEqual(res.status_code, 201)
+
+    def test_email_exist(self):
+        """Test user event rsvp"""
+        res = self.test_client().post('/api/events/1/rsvp', data=self.testemailexist)
+        self.assertEqual(res.status_code, 403)
+
+
 class TestUsers(unittest.TestCase):
     """Test user registration and login"""
     def setUp(self):
         self.app = app
         self.user = User()
         self.test_client = self.app.test_client
-        self.createUser = {
+        self.createuser = {
+            'name': u'Tonnie Nesh',
+            'email': u'tonnie.nesh@gmail.com',
+            'password': u'qwerty1'
+        }
+        self.checkuser = {
             'name': u'Mary Jane',
-            'email': u'jane.mary@yahoo.com',
-            'password': u'1234qwerty'
+            'email': u'john.D@gmail.com',
+            'password': u'qwerty1234'
         }
 
     def tearDown(self):
         del self.user
         del self.test_client
+        del self. createuser
+        del self.checkuser
 
-    def test_user_registration(self):
+    def test_user_reg_email_exist(self):
         """Test user Registration"""
-        resp = self.test_client().post('/api/auth/register', data=self.createUser)
+        resp = self.test_client().post('/api/auth/register', data=self.checkuser)
+        self.assertEqual(resp.status_code, 404)
+
+    def test_reg_email_not_exist(self):

```

Comment:
Again here, the docstrings don't help me understand what you are testing as they are the same as those in <b> test_user_reg_email_exist </b>